nav2_velocity_smoother/src/velocity_smoother.cpp

```diff
@@ -129,9 +129,12 @@ VelocitySmoother::on_configure(const rclcpp_lifecycle::State &)
 
   // Setup inputs / outputs
   smoothed_cmd_pub_ = create_publisher<geometry_msgs::msg::Twist>("cmd_vel_smoothed", 1);
```

Comment:
This server needs to publish out a TwistStamped, else this PR is essentially not valuable since the pipeline goes from the controller server -> collision monitor -> velocity smoother -> robot base / drone. The outputs of the velocity smoother + collision monitor also need to be twiststamped

Reply:
I think that necessitates the inversion of the changes in this file. The TwistStamped should be the actual callback implementing the actual behavior (e.g. storing header to publish out). The un-stamped Cb should create a header to meet the API to go into the stamped callback to use the rest of the pipeline & publish back out unstamped

Reply:
Done. I also switched it to a unique pointer instead of a shared pointer. 
6b110684405a58ed8a906acd1da7a3aef58af171